test/plugin/test_out_lm.rb

```diff
@@ -147,5 +147,97 @@ def create_valid_subject
 
       assert_equal expected, result
     end
+
```

Comment:
Nit: We should have tests for deviceless logs in a separate file, so that each feature get's its own file for better readability.